job_templates/package_action_-_ssh_default.erb

```diff
@@ -52,6 +52,29 @@ exit_with_message () {
   exit $2
 }
 
+<% if package_manager == 'zypper' -%>
+handle_zypp_res () {
+  RETVAL=$1
+
+  # See https://github.com/openSUSE/zypper/blob/master/src/main.h
+  declare -A ZYPRES
```

Comment:
For consistency i would call that variable `ZYPP_RES` or even more descriptive `ZYPP_RES_CODES`.